lightning/config/config.go

```diff
@@ -226,6 +230,11 @@ func (cfg *Config) Load() error {
 		return errors.New("invalid config: `mydumper.csv.delimiter` must be one byte long or empty")
 	}
 
+	cfg.TiDB.SQLMode, err = mysql.GetSQLMode(cfg.TiDB.StrSQLMode)
```

Comment:
`cfg.TiDB.SQLMode` will always assign here, so I think initialize `SQLMode` in L168 is redundancy.